netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -47,10 +47,22 @@
         // @checkstyle LineLengthCheck (1 line)
         "(?<!\\]\\()(?<!=\")(https?:\\/\\/[a-zA-Z0-9-._~:\\?#@!$&'*+,;=%\\/]+[a-zA-Z0-9-_~#@$&'*+=%\\/])(?![\\w.]*\\]\\()"
     );
+    /**
+     * Pattern to detect lines which should have a line break on the end.
+     * We look for lines which have less then two spaces on their end because
+     * TxtMark automatically puts <br/> if the line has two or more spaces
+     * on its end and we should skip a line that starts with four or more
+     * spaces because it's a code block in markdown.
+     */
+    private static final Pattern NEW_LINE = Pattern.compile(
+        "^ {0,3}(\\S|(\\S.*\\S)) ?$", Pattern.MULTILINE
```

Comment:
@mkordas it doesn't make sense, TxtMark doesn't add `<br/>` to lines that contain only spaces